packages/@o3r/create/src/index.ts

```diff
@@ -191,18 +205,33 @@ const prepareWorkspace = (relativeDirectory = '.', projectPackageManager = 'npm'
   writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2));
 
   if (projectPackageManager === 'yarn') {
-    exitProcessIfErrorInSpawnSync(2, spawnSync(runner, ['set', 'version', argv['yarn-version'] || 'stable'], {
-      stdio: 'inherit',
-      shell: true,
-      cwd
-    }));
+    const yarnVersion = quote(argv['yarn-version']);
```

Comment:
why `quote` is required here?

Reply:
It was to fix the security issue raised by @cpaulve-1A https://github.com/AmadeusITGroup/otter/pull/1794#discussion_r1616131180